pkg/kubernetes/types.go

```diff
@@ -56,6 +56,10 @@ type InformerManager struct {
 
 	// ConfigMap informer
 	configMapInformer cache.SharedInformer
+
+	// namespaceInformer informer
+	namespaceInformer cache.SharedInformer
```

Comment:
Can we place `namespaceInformer` and `namespaceSynced` together? Just like every other resource in this struct

Reply:
done